.github/ISSUE_TEMPLATE/portfolio-review.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Portfolio Review
+about: Please review my Portfolio on a live stream
+title: '[REVIEW]'
+labels: portfolio review
+assignees: ''
+---
+
+**Please ensure that you dont have any personal details in your Portfolio that you dont want to share publicly.**
+
+Portfolio link: 
+
+You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)
```

Comment:
```suggestion
You can also join the discord community [here](http://discord.eddiehub.org)
```
Maybe we can use the sub-domian link here, since that would almost always work.